faq/faq.rst

```diff
@@ -0,0 +1,76 @@
+.. _faq:
+
+Frequently Asked Questions
+##########################
+
+
+How do I add nodes to an existing Citus cluster?
+------------------------------------------------
+
+You can add nodes to a Citus cluster by adding the DNS host name and port of the new node to the pg_worker_list.conf configuration file in the data directory. After adding a node to an existing cluster, it will not contain any data (shards). Citus will start assigning any newly created shards to this node. To rebalance existing shards from the older nodes to the new node, the Citus Enterprise edition provides a shard rebalancer utility. You can find more information about shard rebalancing in the :ref:`cluster_management` section
+
+How do I change the shard count for a hash partitioned table?
+-------------------------------------------------------------
+
+Optimal shard count is related to the total number of cores on the workers. Citus partitions an incoming query into its fragment queries which run on individual worker shards. Hence the degree of parallelism for each query is governed by the number of shards the query hits. To ensure maximum parallelism, you should create enough shards on each node such that there is at least one shard per CPU core.
+
+We typically recommend creating a high number of initial shards, e.g. 2x or 4x the number of current CPU cores. This allows for future scaling if you add more workers and CPU cores.
+
+How does citus hadle failure of a worker node?
+----------------------------------------------
+
+If a worker node fails during e.g. a SELECT query, jobs involving shards from that server will automatically fail over to replica shards located on healthy hosts. This means intermittent failures will not require restarting potentially long-running analytical queries, so long as the shards involved can be reached on other healthy hosts.
+You can find more information about Citus' failure handling logic in :ref:`dealing_with_node_failures`.
+
+How does Citus handle failover of the master node?
+--------------------------------------------------
+
+As the Citus master node is similar to a standard PostgreSQL server, regular PostgreSQL synchronous replication and failover can be used to provide higher availability of the head node. Many of our customers use synchronous replication in this way to add resilience against head node failure. You can find more information about handling :ref:`master_node_failures`.
```

Comment:
I had one more edit. Instead of 'head node' towards the end, can we rename that to 'master node'? (Looks like I updated FAQ, but forgot mentioning this edit.